CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+## [2.0.2] - 2024-09-19
+
```

Comment:
```suggestion
* [PR-12](https://github.com/ITK-Leantime/project-overview/pull/14)
  * Removed additional border-radiusses
  * Format javascript dates to force /'s instead of .'s.
```